packages/web-runtime/src/pages/account.vue

```diff
@@ -152,113 +171,202 @@ export default defineComponent({
-      )
 
-      const newSetting = { identifier: accountSettingIdentifier, value }
-      store.commit('SET_SETTINGS_VALUE', newSetting)
-      setCurrentLanguage({ language, languageSetting: newSetting })
+        return value
+      } catch (e) {
+        throw e
+      }
     }
-    const accountEditLink = computed(() => {
-      return store.getters.configuration?.options?.accountEditLink
-    })
 
-    const groupNames = computed(() => {
-      if (unref(useCapabilitySpacesEnabled())) {
-        return unref(user)
-          .groups.map((group) => group.displayName)
-          .join(', ')
+    const updateSelectedLanguage = async (option) => {
+      try {
+        const value = await saveValue({
+          identifier: 'language',
+          valueOptions: { listValue: { values: [{ stringValue: option.value }] } }
+        })
+        selectedLanguageValue.value = option
+        setCurrentLanguage({
+          language,
+          languageSetting: {
+            identifier: {
+              extension: 'ocis-accounts',
+              bundle: 'profile',
+              setting: 'language'
+            },
+            value
+          }
+        })
+        store.dispatch('showMessage', {
+          title: $gettext('Language was saved successfully.')
+        })
+      } catch (e) {
+        console.error(e)
+        store.dispatch('showMessage', {
+          title: $gettext('Saving language failed…'),
+          status: 'danger'
+        })
       }
+    }
 
-      return unref(user).groups.join(', ')
-    })
+    const updateDisableEmailNotifications = async (option) => {
+      try {
+        await saveValue({
+          identifier: 'disable email notifications',
+          valueOptions: { boolValue: !option }
+        })
+        disableEmailNotificationsValue.value = option
+        store.dispatch('showMessage', {
+          title: $gettext('Email notifications preference saved successfully.')
+        })
+      } catch (e) {
+        console.error(e)
+        store.dispatch('showMessage', {
+          title: $gettext('Unable to save email notifications preference…'),
+          status: 'danger'
+        })
+      }
+    }
 
-    const logoutUrl = computed(() => {
-      return configurationManager.logoutUrl
-    })
+    onMounted(async () => {
+      if (unref(isSettingsServiceSupported)) {
+        await loadBundlesList.perform()
+        await loadValuesList.perform()
+
+        const languageConfiguration = unref(valuesList)?.find(
+          (cV) => cV.identifier.setting === 'language'
+        )
+        selectedLanguageValue.value = languageConfiguration
+          ? unref(languageOptions)?.find(
+              (lO) => lO.value === languageConfiguration.value?.listValue?.values?.[0]?.stringValue
+            )
+          : unref(languageOptions)?.find((o) => o.default)
+
+        const disableEmailNotificationsConfiguration = unref(valuesList)?.find(
+          (cV) => cV.identifier.setting === 'disable email notifications'
```

Comment:
I hope that my wish to change this to `disable-email-notifications` will be respected by the backend team 🙈